src/vs/editor/common/codecs/baseToken.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Why are these files in `editor/`?

Reply:
They rely on the `Range` object which is part of the `editor` 🤷 That is the only dependency that keeps it under the `/editor` folder.

Reply:
Are you suggesting moving them under `/workbench`? I've put them to lowest possibile layer I believe.

Reply:
I see, because otherwise they could be in `base`? Let's keep it in `workbench` for now. I think it makes more sense there because it's not used by the editor, it's only using an editor type, which lots of code does, but it's not part of the text editor component.